packages/babel-preset-react/src/index.js

```diff
@@ -9,24 +9,21 @@ import transformReactPure from "@babel/plugin-transform-react-pure-annotations";
 export default declare((api, opts) => {
   api.assertVersion(7);
 
-  let { pragma, pragmaFrag } = opts;
-
   const {
     pure,
+    pragma,
+    pragmaFrag,
+    development = false,
     throwIfNamespace = true,
-    runtime = "classic",
+    runtime = "automatic",
     importSource,
   } = opts;
 
-  // TODO: (Babel 8) Remove setting these defaults
-  if (runtime === "classic") {
-    pragma = pragma || "React.createElement";
-    pragmaFrag = pragmaFrag || "React.Fragment";
+  if (typeof development !== "boolean") {
+    throw new Error(
+      "@babel/preset-react 'development' option must be a boolean.",
+    );
   }
-
-  // TODO: (Babel 8) Don't cast these options but validate it
-  const development = !!opts.development;
```

Comment:
Un-casting these options has been covered in #10927.

Reply:
Ah, I see.

Reply:
I have removed my changes.